cms/static/sass/elements/_feature-anouncement.scss

```diff
@@ -0,0 +1,43 @@
+// studio - elements - announcement bar
+// ====================================
+
+.announcement-bar {
+    position: relative;
+    background: #03c7e8;
```

Comment:
nit: it is preferable to use variables

Reply:
It might be some kind of accent color but I didn't find any matches. 
Let's wait for the review on the upstream repo.